jest/genConfig.js

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
Could we make this filename `gen-config.js` I don't think it needs to be camel cased.